src/nn/accumulator.c

```diff
@@ -16,7 +16,10 @@
 
 #include "accumulator.h"
 
+#if defined(__x86_64__) || defined(_M_X64)
 #include <immintrin.h>
+#endif
+

```

Comment:
I actually don't think I use anything from immintrin in this file, so this include can just be removed.